frappe/desk/doctype/number_card/number_card.py

```diff
@@ -165,7 +165,7 @@ def get_result(doc, filters, to_date=None):
 	)
 	number = res[0]["result"] if res else 0
 
-	return cint(number)
+	return flt(number)
```

Comment:
@fadilsid just remembered something, we should use system precision here to reduce decimal numbers. It will be annoying to see `120.000000000000001` or something like that?

Reply:
> @fadilsid just remembered something, we should use system precision here to reduce decimal numbers. It will be annoying to see `120.000000000000001` or something like that?

Would this work

```return round(flt(number), 3)```

Reply:
`flt(number, precision)`